pkg/services/annotations/annotationsimpl/annotations_test.go

```diff
@@ -1,368 +0,0 @@
-package annotationsimpl
```

Comment:
Needs grafana to be running now - moved to pkg/tests/api/annotations/annotations_accesscontrol_test.go